spec/features/users/user_signs_up_spec.rb

```diff
@@ -43,6 +43,24 @@
     expect(page).to have_content(error_message('steamid.invalid'))
   end
 
+  scenario 'with out of range Steam ID' do
+    within registration_form do
+      fill_form(:user, user.slice(*sign_up_attributes).merge({ steamid: "0:0:2147483648" }))
+      click_button submit(:user, :create)
+    end
+
+    expect(page).to have_content(error_message('steamid.invalid'))
+  end
+
+  scenario 'with nil Steam ID' do
+    within registration_form do
+      fill_form(:user, user.slice(*sign_up_attributes).merge({ steamid: nil }))
+      click_button submit(:user, :create)
+    end
+
+    expect(page).to have_content(error_message('steamid.invalid'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.